docs/install-machine.md

```diff
@@ -20,7 +20,7 @@ installation</a> instructions.
 
 If you only want Docker Machine, you can install the Machine binaries (the
 latest versions of which are located at
-https://github.com/docker/machine/releases/) directly by following the
+https://github.com/docker/machine/releases/ ) directly by following the
```

Comment:
Not sure why we need this? https://github.com/docker/machine/blob/master/docs/install-machine.md looks ok